docs/src/layouts/links.js

```diff
@@ -29,6 +29,13 @@ const ComponentsLinks = [
     category: 'Components',
     sort: true,
     links: [
+      {
+        id: 'GRAPHIC_PROMO',
+        route: routes.GRAPHIC_PROMO,
+        children: 'Graphic promo',
+        tags: ['web'],
+        keywords: ['advert'],
```

Comment:
could you also add `graphic promo` as an option here too as these keywords work to power the search bar on the website